rules/index.go

```diff
@@ -0,0 +1,17 @@
+package rules
+
+import "fmt"
+
+func NotNull(fieldName string, fieldValue interface{}, locale string, ve map[string][]string) map[string][]string {
+	if fieldValue == nil || fieldValue == "" {
```

Comment:
in golang you can't determined `interface{}` whether is nil or not by using `fieldValue == nil`. the `interface` in golang store both type and value, only both of them are nil then `fieldValue == nil` is true.